frontend/src/metabase/dashboard/hoc/DashboardControls.jsx

```diff
@@ -24,7 +24,7 @@ export default ComposedComponent =>
 
       state = {
         isFullscreen: false,
-        isNightMode: false,
+        theme: null,
```

Comment:
Store `theme` instead of`isNightMode`, as a boolean can only represent 2 themes, but we now have 3 themes with this change
- light (or `null`)
- dark
- transparent